LEGO1/lego/legoomni/src/actors/skateboard.cpp

```diff
@@ -17,17 +20,44 @@ SkateBoard::SkateBoard()
 	NotificationManager()->Register(this);
 }
 
-// STUB: LEGO1 0x10010000
+// FUNCTION: LEGO1 0x1000ff80
+SkateBoard::~SkateBoard()
+{
+	ControlManager()->Unregister(this);
+	NotificationManager()->Unregister(this);
+}
+
+// FUNCTION: LEGO1 0x10010000
 MxResult SkateBoard::Create(MxDSAction& p_dsAction)
 {
-	// TODO
-	return SUCCESS;
+	MxResult result = IslePathActor::Create(p_dsAction);
+	if (result == SUCCESS) {
+		this->m_world = CurrentWorld();
```

Comment:
Generally we shouldn't be using `this->` unless needed. You can remove it. There are other instances in the PR where it should be removed as well. Thanks!